packages/react-components/src/locales/index.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Locale } from '../localization/';
+import _en_US from './translated/en_US.json';
+import _de_DE from './translated/de_DE.json';
+import _ar_SA from './translated/ar_SA.json';
+import _fr_FR from './translated/fr_FR.json';
+
+/** Locale for English (US) */
+export const en_US: Locale = { strings: _en_US };

```

Comment:
As said in Miguel PR around this, wondering if it will be tter to have a single object gathering all those locales

Reply:
Actually, it's better to keep these constants separate. I want to land a test in the treeshaking package that if an app only imports en_US, none of the other locales / JSON data is loaded. This means that Contoso can simply import the handful of locales they want without paying the cost of a larger app bundle from the unused locales.

Reply:
I think Aline means we can simply get rid of the `Locale` object since it only has one prop and simply use `ComponentStrings` as the context for LocalizationProvider. I like that idea too. But we can do that in another PR.